hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -121,17 +120,34 @@ public void testShutdown() {
     public void testConvertToLetter() {
         HazelcastInstance hz = createHazelcastInstance();
         Node node = getNode(hz);
+//        PhoneHome phoneHome = new PhoneHome(node);
+//        MetricsCollectorImpl metricsCollector = new MetricsCollectorImpl(node);
+//        assertEquals("A", metricsCollector.convertToLetter(4));
+//        assertEquals("B", metricsCollector.convertToLetter(9));
+//        assertEquals("C", metricsCollector.convertToLetter(19));
+//        assertEquals("D", metricsCollector.convertToLetter(39));
+//        assertEquals("E", metricsCollector.convertToLetter(59));
+//        assertEquals("F", metricsCollector.convertToLetter(99));
+//        assertEquals("G", metricsCollector.convertToLetter(149));
+//        assertEquals("H", metricsCollector.convertToLetter(299));
+//        assertEquals("J", metricsCollector.convertToLetter(599));
+//        assertEquals("I", metricsCollector.convertToLetter(1000));
```

Comment:
Please don't comment out assertions. Can you please  restore them after you extracted `convertToLetter()` into `MetricsCollector` ?